lldb/include/lldb/Core/Module.h

```diff
@@ -437,6 +438,21 @@ class Module : public std::enable_shared_from_this<Module>,
   ///     TypeMap::InsertUnique(...).
   void FindTypes(const TypeQuery &query, TypeResults &results);
 
+  /// Finds imported declarations whose name match \p name.
+  ///
+  /// \param[in] name
+  ///     The name to search the imported declaration by.
+  ///
+  /// \param[in] results
+  ///     Any matching types will be populated into the \a results object.
+  ///
+  /// \param[in] find_one
+  ///     If set to true, the search will stop after the first imported
+  ///     declaration is found.
+  void FindImportedDeclarations(ConstString name,
+                                std::vector<ImportedDeclaration> &results,
```

Comment:
Unless you need to append to the same vector multiple times, it should probably be a return type.

Reply:
You do append to the vector multiple times as part of the implementation (for example on `SymbolFileDWARFDebugMap`). This is also how all of the other query APIs in Module and SymbolFile do it, so I'd rather keep it this way to stay consistent.